src/shadowfax.py

```diff
@@ -224,10 +222,10 @@ async def shadowfax(message, guid, logger):
         num_paths += 1
         fits_constraint = False
         for curie in path:
-            if curie not in [source_node, target_node] and curie in curie_info.keys():
+            if curie not in [source_node, target_node]:
```

Comment:
Why was curie_info taken out here? Do we not want normalized curies?

Reply:
Strider should be giving us back normalized curies anyway. That was old code that prevented paths that went through nodes that were not selected previously in node selection. With the decision to also add paths that are may come from nodes surfaced from support graphs, that check must be removed. We just add a similar check based on the knowledge graph a bit lower.